### PR TITLE
impl: read service config

### PR DESCRIPTION
**Pull request by julieqiu**
Fixes https://github.com/googleapis/google-cloud-rust/issues/38
Updates https://github.com/googleapis/google-cloud-rust/issues/42

### CODE REVIEW

File: generator/internal/genclient/translator/protobuf/protobuf.go [hunk 1]
```diff
@@ -100,11 +102,12 @@ func makeAPI(req *pluginpb.CodeGeneratorRequest) *genclient.API {
 			}
 			state.ServiceByID[service.ID] = service
 			for _, m := range s.Method {
-				method := &genclient.Method{}
-				method.HTTPInfo = parseHTTPInfo(m.GetOptions())
-				method.Name = m.GetName()
-				method.InputTypeID = m.GetInputType()
-				method.OutputTypeID = m.GetOutputType()
+				method := &genclient.Method{
```
Comment (coryan) [inline generator/internal/genclient/translator/protobuf/protobuf.go:105]: This change seem unrelated, but meh.

File: generator/internal/sample/sample.go
Comment (coryan) [inline generator/internal/sample/sample.go:1]: This file seems unrelated.

Comment (julieqiu) [reply generator/internal/sample/sample.go:1]: This is used here: https://github.com/googleapis/google-cloud-rust/pull/83/files/c104b47d57bc267315065c3ec268f7b98b5b08d3#diff-1ccc675e06a72e303cb180fa7d4b0c8ab66ec09d33be3f2571cb04e2779da506R34

File: generator/internal/genclient/translator/protobuf/protobuf.go (hunk 1 above)
Comment (julieqiu) [reply generator/internal/genclient/translator/protobuf/protobuf.go:105]: This was more for readability, but happy to put in a separate PR in the future

**Review by coryan — APPROVED** [review]

File: generator/internal/sample/sample.go
Comment (coryan) [reply generator/internal/sample/sample.go:1]: Thanks!
